### PR TITLE
Move from zip to lzma compressed archive when building Linux packages

### PR DESCRIPTION
**Pull request by jordisayol**

### CODE REVIEW

File: linux/dmd_rpm.sh [hunk 1]
```diff
@@ -134,7 +134,7 @@ do
 
 
 		# download zip file if not exist
-		if ! $(unzip -c $DESTDIR"/"$ZIPFILE &>/dev/null)
+		if ! $(tar --lzma -tf $DESTDIR"/"$ZIPFILE &>/dev/null)
```
Comment (MartinNowak) [inline linux/dmd_rpm.sh]: `tar -Jtf`?

**Comment by MartinNowak** [comment]
Are these scripts used outside of our build system?

**Comment by MartinNowak** [comment]
It will also be incompatible with #140 b/c the .tar.xz packages are build from the .zip not the other way round. This could prolly be changed though.

**Comment by jordisayol** [comment]
I thought that zip archives will be replaced by "*.tar.xz" and "*.7z" for the next release. My mistake. I close this PR.

**Comment by MartinNowak** [comment]
> I thought that zip archives will be replaced by ".tar.xz" and ".7z" for the next release. My mistake. I close this PR.

We'll have them in addition, maybe deprecate them at some point, but not soon.
It would still be cleaner to generate the zips from the tar.xz, instead of doing it vice versa.

**Pull request closed without merging**